CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# HERMES development workflow
+
+## Preamble
+
+> **Branching is cheap!**
+>
+> Work in small packets, put up small pull requests!
+>
+> Aim for quick turnaround times!
+
+## Branching
+
+We loosely follow a mixture of [GitHubFlow](https://docs.github.com/en/get-started/quickstart/github-flow) and [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/):
+
+`main`
+    - Stable branch
+    - Merges come only from `develop` or a hotfix branch (i.e., when something needs to be fixed in "production")
+
+`develop`
+    - Unstable development branch
+    - Last "stable" is tagged: tag before attempting to break something
+
+`feature/<describe-feature>` (including issue id when exists: `feature/62-fix-broken-thing`)
+    - Branch from last tag on `develop`
+    - Feature branches may pick up "older" branches if they are not to complicated
```

Comment:
**question**: What does that mean? Merge? Rebase first? Cherrypick?

Reply:
IMHO, cherrypicking is a last resort, and often a sign that something is broken in the code or the workflow or the scope of the work. Merging is verbose, which is good i think.

Also need to fix that typo.

Reply:
OK.